.circleci/config.yml

```diff
@@ -427,6 +427,17 @@ jobs:
                 docker image prune -a --force --filter "until=${time}h"
               fi
            done
+
+      - run:
+         name: Login into gcr
+         command: |
+            echo ${GCLOUD_SERVICE_KEY} | docker login -u _json_key --password-stdin https://gcr.io
+
+      - run:
+         name: Build docker builder image
+         command: |
+                docker build -t ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} --target builder .
```

Comment:
Have to check if it works, but the `DOCKER_BRANCH_TAG`  is only generated a step later, right?

Reply:
Missed that. Thought it is a default circleci variable 
And it isnt working, because build failed